libcloud/compute/drivers/openstack.py

```diff
@@ -2196,20 +2216,26 @@ def _to_volume(self, api_node):
 
         return StorageVolume(
             id=api_node['id'],
-            name=api_node['displayName'],
+            name=api_node.get('name', api_node.get('displayName')),
             size=api_node['size'],
             state=state,
             driver=self,
             extra={
-                'description': api_node['displayDescription'],
+                'description': api_node.get('description',
+                                            api_node.get('displayDescription')
```

Comment:
same thing here, shouldn't the displayDescription be leading for backwards compatibility?

Reply:
I agree, I will change it.